src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -175,9 +175,10 @@ GoogleRecaptchaVerifier googleRecaptchaVerifier(
     BasicHttpClient basicHttpClient(
             Vertx vertx,
             @Value("${http-client.max-pool-size}") int maxPoolSize,
-            @Value("${http-client.connect-timeout-ms}") int connectTimeoutMs) {
+            @Value("${http-client.connect-timeout-ms}") int connectTimeoutMs,
+            @Value("${http-client.use-compression:false}") boolean useCompression) {
```

Comment:
Usually we try to avoid in-annotation configuration default params and preferably use default config file `src/main/resources/application.yaml`.
Please, check the same for `circuitBreakerSecuredHttpClient` bean.

Reply:
Sure